packages/babel-parser/test/fixtures/experimental/import-assertions-with-keyword/linebreak-before-with/input.js

```diff
@@ -0,0 +1,2 @@
+import "foo"
+with { type: "json" }
```

Comment:
The current spec disallows such case: https://tc39.es/proposal-import-attributes/#prod-ImportDeclaration